packages/client/src/views/SysAdmin/Config/Integrations/Integrations.tsx

```diff
@@ -10,10 +22,30 @@ import { Frame } from '@opencrvs/components/lib/Frame'
 import { Navigation } from '@client/components/interface/Navigation'
 import { Header } from '@client/components/Header/Header'
 import { Plus, VerticalThreeDots } from '@client/../../components/lib/icons'
-import { Pill, ToggleMenu } from '@client/../../components/lib'
-import { constantsMessages } from '@client/i18n/messages'
+import { WebhookOption } from '@client/utils/gateway'
+
+import {
+  Alert,
+  CheckboxGroup,
+  Divider,
+  FormTabs,
+  InputField,
+  Link,
+  Pill,
+  Select,
+  Spinner,
+  Stack,
+  TextInput,
+  ToggleMenu
+} from '@client/../../components/lib'
```

Comment:
lets change `@client/../../components/lib` to `@opencrvs/components/lib` in all places where it's used in this file